### PR TITLE
Update the README file

### PR DESCRIPTION
**Pull request by banuakman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,39 @@
...
+WHEN a user chooses to search restaurants, they are prompted for their location. If they share location 
+A MAP APPEARS with the results near their current location.
+Markers with pop up windows as well as an easy to view lit allow the user to be directed to website of their choosing. 
+If the user decides not to share location. they can manually search the map in a search map by inputting a query and location. 
+
+ON ALL THE WINDOWS, there is an option to randomly generate three more random dishes.
+```
+
+## Technologies Used
+This application implements vanilla javaScript as well a jQuery to dynamically traverse DOM elements on the page. The deployed webpage utilizes the the Spoonacular and Google Maps APIs to retrieve the recipes restaurants and location data that displayed to the user. the Foundation CSS framework was used to style the responsive layout.
```
Comment (angelinama) [inline README.md:27]: as well as

File: README.md [hunk 2]
```diff
@@ -1,9 +1,39 @@
...
+## Motivation
+The What's for Dinner Application is built to provide the user with an easy to navigate dinner selection process
+With the ever looming question, "What's for dinner?" causing headaches and wasting time. The app narrows ones choices by delivering 3 randomly generated options with clickable images. 
 
-This application implements vanilla javaScript as well a jQuery to dynamically traverse DOM elements on the page. The deployed webpage utilizes the the Spoonacular and Google Maps APIs to retrieve the recipes restaurants and location data that displayed to the user. the Foundation CSS framework was used to style the page.
+## User Story
+```
+GIVEN 3 randomly generated options with clickable images.
+WHEN the user presses What is for Dinner? button
```
Comment (angelinama) [inline README.md:10]: "What is For Dinner" button

File: README.md [hunk 3]
```diff
@@ -1,9 +1,39 @@
...
+THEN user presses select an dish or randomly generate another option
```
Comment (angelinama) [inline README.md:11]: clicks image to select

Comment (angelinama) [reply README.md:11]: generate anther three options

File: README.md [hunk 4]
```diff
@@ -1,9 +1,39 @@
...
+
+WHEN a dish is selected, a drop down menu appears and the user can then choose to see a recipe or search nearby restaurants for the selected dish. 
+
+WHEN the recipe is chosen,
+SUMMARY and LINK to the recipe appears, as well a few links to similar recipes. 
+
+WHEN a user chooses to search restaurants, they are prompted for their location. If they share location 
+A MAP APPEARS with the results near their current location.
+Markers with pop up windows as well as an easy to view lit allow the user to be directed to website of their choosing.
```
Comment (angelinama) [inline README.md:20]: list

File: README.md (hunk 1 above)
Comment (angelinama) [inline README.md:27]: there are two "the" before Spoonacular

**Review by benrgross — APPROVED** [review]

**Pull request merged**